shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/test/java/org/apache/shardingsphere/distsql/parser/api/DistSQLStatementParserEngineTest.java

```diff
@@ -42,6 +44,12 @@
     
     private static final String RDL_DROP_RESOURCE = "DROP RESOURCE ds_0,ds_1";
     
+    private static final String RDL_CREATE_SHARDING_TABLE_RULE = "CREATE SHARDING TABLE RULE t_order (" 
+            + "RESOURCES(ms_group_0,ms_group_1)," 
+            + "SHARDING_COLUMN=order_id," 
+            + "TYPE(NAME=hash_mod,PROPERTIES(\"sharding-count\"=4))," 
```

Comment:
'sharding-count' is possible to be used as well,  right?

Reply:
UT has been added for `sharding-count` and other properties.